justfile

```diff
@@ -74,3 +74,6 @@ bench path:
 # example: just clippy dns
 clippy package="...":
   bazel build //src/rust/{{package}} --config=lint
+
+prepare-ubuntu:
+  sudo apt-get install -y --no-install-recommends libc++abi1-18 libc++1-18 libc++-18-dev lld-18
```

Comment:
More packages will be needed – Bazelisk for Bazel itself, the readme also lists several other packages (e.g. python3 for V8, tcl for sqlite3)

Reply:
You're right. Can you make suggestions to the code?

Reply:
I didn't think that far ahead TBH. Readme suggests `python3`, `python3-distutils`, and `tcl8.6`, note that there's no Ubuntu bazelisk package though